src/Zicht/Bundle/FrameworkExtraBundle/Command/RepairNestedTreeCommand.php

```diff
@@ -44,31 +46,29 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
+        $logger = $this->createStdLogger($output, 'console');
```

Comment:
In deze bestanden gebruiken we alleen `writeln` maar hoe werkt dit samen met outputhelpers zoals `Table` https://symfony.com/doc/current/components/console/helpers/table.html? Moet je dan bijv. zelf de `render()` call conditioneel doen obv de verbosity?

Reply:
De helpers geven doorgaan een string terug, maar die string kan ook newlines verwachten.  Als je een tabel in een log entry zet, dan kan dat wel, maar echt mooie output zal het niet worden.  Als je toch een tabel oid wilt loggen, dan zul je de tabel string moeten splitten op newline, en dan regel voor regel moeten loggen.

Niet ideaal, maar $output->write(...) gaat helaas altijd naar stderr.  Dus als je verwacht dat het command gebruikt gaat worden middels een cronjob, dan kun je dat dus niet meer gebruiken.

Hetzelfde probleem komen we ook tegen met progress indicators, die kun je ook niet loggen.  Dus niet ideaal, maar een stuk beter dan die 30k error mailtjes die we nu hebben van cronjobs.